### ISSUE TITLE
lines.length > 0

### ISSUE THREAD

**Issue by c0c4i** [RomanBase/hand_signature#14]
Hello, I'm using this package but I have this problem below while writing a signature.
```
'package:hand_signature/src/utils.dart': Failed assertion: line 369 pos 12: 'lines.length > 0': is not true.
The relevant error-causing widget was HandSignaturePainterView
```
This is my code to show widget
```dart
Container(
  width: double.infinity,
  height: 250,
  child: HandSignaturePainterView(
    control: control,
    decoration: ... ,
    type: SignatureDrawType.line,
    width: 3,
  ),
)
```
And this is style of `HandSignatureControl`
```dart
final HandSignatureControl control = new HandSignatureControl(
    threshold: 0.01,
    smoothRatio: 0.65,
    velocityRange: 2.0,
  );
```

Do you have any solution?

**Comment by RomanBase** [comment]
Hi, can you provide more info when this error occurs (during drawing, on touch up, export) ?

**Comment by c0c4i** [comment]
Yep sorry, it occurs during drawing

**Comment by RomanBase** [comment]
Solved in `2.1.0+2`
